plugins/JsonRPCServer.js

```diff
@@ -19,6 +19,54 @@ let serverRPC = null;
 let server = null;
 let database = null;
 
+async function generateStatus() {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const result = {};
+      // retrieve the last block of the sidechain
+      const block = await database.getLatestBlockMetadata();
+
+      if (block) {
+        result.lastBlockNumber = block.blockNumber;
+        result.lastBlockRefHiveBlockNumber = block.refHiveBlockNumber;
+        result.lastHash = block.hash;
+      }
+
+      // get the Hive block number that the streamer is currently parsing
+      const res = await ipc.send(
+        { to: STREAMER_PLUGIN_NAME, action: STREAMER_PLUGIN_ACTION.GET_CURRENT_BLOCK },
+      );
+
+      if (res && res.payload) {
+        result.lastParsedHiveBlockNumber = res.payload;
+      }
+
+      // get the version of the SSC node
+      result.SSCnodeVersion = packagejson.version;
+
+      // gets the domain of the SSC node
+      result.domain = config.domain;
+
+      // get the ssc chain id from config
+      result.chainId = config.chainId;
+
+      // get light node config of the SSC node
+      result.lightNode = config.lightNode;
+      result.blocksToKeep = config.blocksToKeep;
```

Comment:
I think blocksToKeep should only be added to this status info if lightNode = true (since otherwise it doesn't apply and so might be confusing if it's present).